src/views/Home/components/Hero/Hero.scss

```diff
@@ -6,13 +6,36 @@
   overflow: hidden;
   min-height: 100%;
 
+  @mixin flare {
+    position: absolute;
+    z-index: -3;
+    width: 110vw;
+    height: calc(100%);
+    max-height: 1900px;
+    top: calc(-20vh);
+    margin-right: calc(-4vw - 5vh);
+
+    @include responsiveness.respond(tab-port) {
+      top: calc(-20vh - 20vw);
+      margin-right: calc(-35vw - 5vh);
+    }
+
+    @include responsiveness.respond(phone) {
+      visibility: hidden;
+    }
+  }
+
   &__container {
     display: flex;
+    position: relative;
     flex-wrap: nowrap;
-    margin: auto;
-    max-width: 1800px;
+    margin: 0;
+    width: 100%;
     padding-top: calc(65px + 10vh);
+    padding-bottom: calc(65px + 10vh);
```

Comment:
Maybe decrease this so there's not as much of a noticeable gap between sections.

Reply:
@tim-nguyen-cs since this component is the hero, I tried to have the element occupy most of the viewport. our past websites had the hero occupy the entire viewport, but I haven't integrated that for this website, given this year's constraints on time spent designing & boilerplating

Reply:
Ohh, okay ~ that makes sense! In that case, we'd definitely need something to hint that there's more content "under the fold," but that also sounds like a good plan, aha.